packages/components/src/components/scrollable/scrollable.ts

```diff
@@ -0,0 +1,316 @@
+                <slot name="icon-end">
+                  <sd-icon library="system" name="chevron-down"></sd-icon>
+                </slot>
+              </button>
+            </div>
+          `
+        : null}
+      ${this.shadows && this.isScrollHorizontalEnabled && this.canScroll.left
+        ? html`<div part="shadow-left" class="${scrollShadowClasses} left top-0 left-0 w-[6px] h-full"></div>`
+        : null}
+      ${this.shadows && this.isScrollHorizontalEnabled && this.canScroll.right
+        ? html`<div part="shadow-right" class="${scrollShadowClasses} right top-0 right-0 w-[6px] h-full"></div>`
+        : null}
+      ${this.shadows && this.isScrollVerticalEnabled && this.canScroll.up
+        ? html`<div part="shadow-top" class="${scrollShadowClasses} top top-0 left-0 w-full h-[6px]"></div>`
+        : null}
+      ${this.shadows && this.isScrollVerticalEnabled && this.canScroll.down
+        ? html`<div part="shadow-bottom" class="${scrollShadowClasses} bottom bottom-0 left-0 w-full h-[6px]"></div>`
+        : null}
+    `;
+  }
+
+  static styles = [
+    SolidElement.styles,
+    unsafeCSS(InteractiveStyles),
+    componentStyles,
+    css`
+      :host {
+        --gradient-color: rgba(255, 255, 255, 0);
+        --gradient: var(--gradient-color) 0%, #fff 80%, #fff 100%;
+
+        @apply flex relative overflow-hidden;
+      }
+
+      .hide-scrollbars {
+        scrollbar-width: none; /* Firefox */
+        -ms-overflow-style: none; /* IE 10+ */
+        &::-webkit-scrollbar {
+          @apply w-0 h-0 bg-transparent;
+        }
+      }
+
+      /* Default state: scrollbar is transparent */
+      .scroll-container::-webkit-scrollbar-thumb {
+        @apply bg-transparent;
+      }
+
+      /* On hover: apply custom styles to the scrollbar thumb */
+      .scroll-container:hover::-webkit-scrollbar-thumb {
+        background: var(--background-neutral-400, #c3c3c3);
+        @apply rounded-sm;
+      }
+
+      /* Apply width and height to the scrollbar itself */
+      .scroll-container::-webkit-scrollbar {
+        @apply w-1 h-1;
+      }
+
+      .button-left {
+        background: linear-gradient(270deg, var(--gradient));
+      }
+
+      .button-right {
+        background: linear-gradient(90deg, var(--gradient));
+      }
+
+      .button-top {
+        background: linear-gradient(0deg, var(--gradient));
+      }
+
+      .button-down {
+        background: linear-gradient(180deg, var(--gradient));
+      }
+
+      .scroll-shadow.left {
+        background: linear-gradient(270deg, rgba(24, 24, 24, 0) 50%, rgba(24, 24, 24, 0.4) 100%);
+      }
+
+      .scroll-shadow.right {
+        background: linear-gradient(90deg, rgba(24, 24, 24, 0) 50%, rgba(24, 24, 24, 0.4) 100%);
+      }
+
+      .scroll-shadow.top {
+        background: linear-gradient(0deg, rgba(24, 24, 24, 0) 50%, rgba(24, 24, 24, 0.4) 100%);
+      }
```

Comment:
Can't we jsut get rid of those classes and instead directly direct to the parts?

```css
[part="shadow-left] {
}
```

etc.